static/js/recent_topics.js

```diff
@@ -352,7 +420,100 @@ exports.launch = function () {
         },
     });
     exports.complete_rerender();
-    $("#recent_topics_search").select();
+};
+
+exports.change_focused_element = function (e, input_key) {
+    if (input_key === 'tab') {
+        input_key = 'right_arrow';
+    } else if (input_key === 'shift_tab') {
+        input_key = 'left_arrow';
+    }
+
+    const $elem = $(e.target);
+
+    if ($("#recent_topics_table").find(':focus').length === 0) {
+        // This is a failsafe to return focus back to recent topics overlay,
+        // in case it loses focus due to some unknown reason.
+        set_default_focus();
```

Comment:
@timabbott we expect user to stay focused in Recent Topics Overlay when it's open. We have to remove this when we move Recent Topics to be the default view.

Reply:
Make sense.